### PR TITLE
add organization member removal test

### PR DESCRIPTION
**Pull request by ubergesundheit**
This ticks the last box (Removing a member from an org) in #759 under Organizations

### CODE REVIEW

File: __tests__/Organization.js [hunk 1]
```diff
@@ -249,6 +249,65 @@ describe('Organizations basic', () => {
...
+    );
+
+    // users
+    const usersTable = await findByTestId('org-detail-users-wrapper');
+    expect(usersTable.querySelector('tbody > tr > td').textContent).toBe(
+      orgResponse.members[0].email
+    );
+
+    const removeUserTableButton = usersTable.querySelector(
+      'tbody > tr > td:nth-of-type(3) > button'
```
Comment (axbarsan) [inline __tests__/Organization.js]: Why not use a test id, as the structure might change in the future?

Comment (ograu) [reply __tests__/Organization.js]: +1

Comment (ubergesundheit) [reply __tests__/Organization.js]: The tables on the organization detail component uses an external component. I'll look into adding test-ids

Comment (ubergesundheit) [reply __tests__/Organization.js]: removed querySelector from this and the organization detail view test

**Pull request merged**